template/cordova/lib/prepare.js

```diff
@@ -420,6 +420,11 @@ module.exports.prepare = function (cordovaProject) {
     this._config = updateConfigFilesFrom(cordovaProject.projectConfig,
         this._munger, this.locations);
 
+    // CB-10845 avoid using cached appxmanifests since they could be
+    // previously modififed outside of AppxManifest class
+    // TODO: invalidate only entries that were affected by config munge
+    AppxManifest.purgeCache();
```

Comment:
It is a `_munger` which applies the changes defined by `<config-file>` elements from `plugin.xml`